R/assign.R

```diff
@@ -0,0 +1,243 @@
+#' Derive an SDTM variable
+#'
+#' @description
+#' [sdtm_assign()] is an internal function packing the same functionality as
+#' [assign_no_ct()] and [assign_ct()] together but aimed at developers only.
+#' As a user please use either [assign_no_ct()] or [assign_ct()].
+#'
```

Comment:
We need to add test cases for these functions. We can add it in a new branch.

Code coverage
```R
sdtm.oak Coverage: 67.52%
R/assign.R: 0.00%
R/clear_cache.R: 0.00%
R/hardcode.R: 0.00%
R/oak_id_vars.R: 0.00%
R/zzz.R: 0.00%
R/dtc_problems.R: 55.26%
R/dtc_utils.R: 60.53%
R/ct.R: 79.25%
R/dtc_create_iso8601.R: 89.00%
R/parse_dttm_fmt.R: 91.87%
R/derive_study_day.R: 91.89%
R/dtc_parse_dttm.R: 100.00%
R/recode.R: 100.00%
R/str_split.R: 100.00%
```

Reply:
I've created a new issue for the improvement of code coverage.